### ISSUE TITLE
MenuItem conflicts with Material one

### ISSUE THREAD

**Issue by prateekmedia** [ymrabti/popup_menu_2#2]
Can you Rename MenuItem to something else otherwise it is conflicting with materials MenuItem

**Comment by ymrabti** [comment]
Yup it is, since the last upgrade of flutter, MenuItem its included in the version, so you can use `import 'package:popup_menu_2/popup_menu_item.dart' as popup_menu_item;` and then `popup_menu_item.MenuItem(`
Otherwise a fixing update is coming soon
